modules/vaos/spec/requests/vaos/v2/appointments_spec.rb

```diff
@@ -1437,7 +1454,22 @@ def stub_clinics
                   .and_return(OpenStruct.new(data: []))
                 post '/vaos/v2/appointments/draft', params: draft_params
 
+                # rubocop:disable Layout/LineLength
+                expected_error_detail = 'BackendServiceException: ' \
+                                        '{:detail=>"invalid patientId", ' \
+                                        ':source=>{' \
+                                        ':vamf_url=>#<URI::HTTPS https://api.wellhive.com/care-navigation/v1/appointments>, ' \
+                                        ':vamf_body=>"{\"message\":\"invalid patientId\",\"name\":\"Bad Request\"}\\n", ' \
```

Comment:
Some of this data will make the test brittle, specially when changing the message that is supposed to be free text. Consider testing for type of error and test for contents on a unit test specific to the Service error class. This would apply to other tests here as well. Just a suggestion.